src/components/molecules/PollBox/index.ts

```diff
@@ -0,0 +1 @@
+export { PollBox as default } from "./PollBox";
```

Comment:
Named exports rather than `default` exports:

```suggestion
export { PollBox } from "./PollBox";
```